src/charm.py

```diff
@@ -127,6 +142,32 @@ def flask_layer(self) -> dict:
             },
         }
 
+    def _on_statsd_prometheus_exporter_pebble_ready(self, _event: PebbleReadyEvent) -> None:
+        """Handle the statsd-prometheus-exporter-pebble-ready event."""
+        statsd_container = self.unit.get_container("statsd-prometheus-exporter")
+        statsd_layer = {
+            "summary": "statsd exporter layer",
+            "description": "statsd exporter layer",
+            "services": {
+                "statsd-prometheus-exporter": {
+                    "override": "replace",
+                    "summary": "statsd exporter service",
+                    "user": "nobody",
+                    "command": "/bin/statsd_exporter",
```

Comment:
Similar to the flask layer, should more of this come from an Observer module?

Reply:
This is pebble layer, and this is a statsd prometheus exporter, which is not a part of canonical observability stack., so I think this should belong to the `charm.py`.

Reply:
Statsd seems to be linked to observability even if it isn't directly part of the observability stack? I'm guessing it is running to extract stats to be forwarded to the observability stack? Or is it used for other things as well?

Reply:
The observability stack accepts only prometheus for metrics, the statsd export is for converting statsd metrics to prometheus metrics since the Gunicorn only support statsd natively.

Reply:
Would it make sense to locate the configuration for statsd in the observer module since it seems to be closely related to observability?

Reply:
There's no configuration for statsd, the only thing needed for starting statsd is the statsd pebble layer. Should we put the statsd pebble layer inside the observability module?

Reply:
It would be good to use constants from the observability module to generate the pebble layer, like make the command a constant in the observability module and use the constant to create the observability layer in `charm.py`. Does that make sense?

Reply:
The problem is that the statsd command to start the statsd service is not tied to the observability stack, instead it is based on this resource definition in the `metadata.yaml` file. For different webservers, they might use different exports other than statsd exporter. That's why I think this should be the charm's responsibility for decide the command of the exporter pebble layer instead of observability module.

```yaml
  statsd-prometheus-exporter-image:
    type: oci-image
    description: Prometheus exporter for statsd data
    auto-fetch: true
    upstream-source: prom/statsd-exporter
```